crates/shared/src/univ3_router_api.rs

```diff
@@ -0,0 +1,115 @@
+use crate::solver_utils::deserialize_prefixed_hex;
+use anyhow::{Context, Result};
+use model::u256_decimal;
+use primitive_types::{H160, U256};
+use reqwest::{Client, Url};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Copy, Clone, Serialize)]
```

Comment:
nit: `#[serde(rename_all = "camelCase")]`

Reply:
For this one I felt giving the enum values explicitly was a bit easier to understand.